state/favorites/operations/toggleToFavorite.js

```diff
@@ -0,0 +1,53 @@
+import { createLogic } from 'redux-logic';
+
+import { getSessionId } from '../../session/selectors';
+import {
+  favoritesTotalItems,
+  favoritesCurrentPage
+} from '../selectors';
+import { TOGGLE_TO_FAVORITES } from '../types';
+import {
+  fetchFavorites,
+  toggleToFavoriteSuccess
+} from '../actions';
+
+const toggleToFavoritesLogic = createLogic({
+  type: TOGGLE_TO_FAVORITES,
+
+  async process({ getState, httpClient, action }, dispatch, done) {
+    const { favorite, id, needRefetchList } = action;
+    const state = getState();
+    const sessionId = getSessionId(state);
+
+    try {
+      await httpClient.post(`/account/{account_id}/favorite?session_id=${sessionId}`, {
+        media_type: 'movie', /* eslint-disable-line camelcase */
+        media_id: id, /* eslint-disable-line camelcase */
+        favorite
+      });
+
+      dispatch(toggleToFavoriteSuccess(id, favorite));
+
+      if (needRefetchList) {
```

Comment:
This code is the same as in watchlist logic. Perhaps, it would be better to reuse it. What do you think?

Reply:
You can move this code to some helper and reuse in both operations.

Reply:
Willl be done.